### PR TITLE
fix(wallet): do not prevent re-submission

### PR DESCRIPTION
**Pull request by mkazlauskas**
# Context

#861 made submitTx idempotent, but also broke internal logic of PersonalWallet preventing it to re-submit transactions

# Proposed Solution

ef64b30c7d02b64746344ea3042ae7b96db888f6

### CODE REVIEW

**Review by mirceahasegan — DISMISSED** [review]
Nice catch! 🕵️‍♂️

**Review by mirceahasegan — APPROVED** [review]

File: packages/wallet/test/PersonalWallet/methods.test.ts [hunk 1]
```diff
@@ -462,6 +464,20 @@ describe('PersonalWallet methods', () => {
         expect(await txPending).toEqual(outgoingTx);
       });
 
+      it('attempts to resubmit the tx that is already in flight', async () => {
```
Comment (rhyslbw) [inline packages/wallet/test/PersonalWallet/methods.test.ts:467]:
```suggestion
      it('only tracks the first submission as in-flight when there are multiple submission of the same tx', async () => {
```
Describe the expectation

**Review by rhyslbw — CHANGES_REQUESTED** [review]

File: packages/wallet/test/PersonalWallet/methods.test.ts (hunk 1 above)
Comment (mkazlauskas) [reply packages/wallet/test/PersonalWallet/methods.test.ts:467]: Main assertion in this test is `expect(txSubmitProvider.submitTx).toBeCalledTimes(2);` though :thinking:

Comment (rhyslbw) [reply packages/wallet/test/PersonalWallet/methods.test.ts:467]: Right I see. I'll retract my suggestion

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**